cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TemporalFunctionsTest.scala

```diff
@@ -0,0 +1,1317 @@
+        considerations("At least one parameter must be provided (`duration()` and `duration({})` are invalid).", "There is no constraint on how many of the parameters are provided.", "It is possible to have a _Duration_ where the amount of a smaller unit (e.g. `seconds`) exceeds the threshold of a larger unit (e.g. `days`).", "The values of the parameters may be expressed as decimal fractions.", "The values of the parameters may be arbitrarily large.", "The values of the parameters may be negative.")
+        query(
+          """UNWIND [duration({days: 14, hours:16, minutes: 12}),
+            |   duration({months: 5, days: 1.5}),
+            |   duration({months: 0.75}),
+            |   duration({weeks: 2.5}),
+            |   duration({years: 12, months:5, days: 14, hours:16, minutes: 12, seconds: 70}),
+            |   duration({days: 14, seconds: 70, milliseconds: 1}),
+            |   duration({minutes: 1.5, seconds: 1})] AS d
+            |RETURN d""".stripMargin, ResultAssertions((r) => {
+            //CYPHER_TODO
+          })) {
+          resultTable()
+        }
+      }
+      section("Creating a _Duration_ from a string", "functions-duration-create-string") {
+        p(
+          """`duration()` returns the _Duration_ value obtained by parsing a string representation of a temporal amount.""".stripMargin)
+        function("duration(temporalAmount)", "A Duration.", ("temporalAmount", "A string representing a temporal amount."))
+        considerations("`temporalAmount` must comply with either the <<cypher-temporal-specifying-durations, unit based form or date-and-time based form defined for _Durations_>>.")
+        query(
+          """UNWIND [duration("P14DT16H12M"),
+            |   duration("P5M1.5D"),
+            |   duration("P0.75M"),
+            |   duration("PT0.75M"),
+            |   duration("P2.5W"),
+            |   duration("P12Y5M14DT16H12M70S"),
+            |   duration("P2012-02-02T14:37:21.545")] AS d
+            |RETURN d""".stripMargin, ResultAssertions((r) => {
+            //CYPHER_TODO
+          })) {
+          resultTable()
+        }
+      }
+      section("Computing the _Duration_ between two temporal instants", "functions-duration-computing") {
+        p(
+          """`duration()` has sub-functions which compute the _logical difference_ (in days, months, etc) between two temporal instant values:
+            |
+            |* `duration.between(a, b)`: Computes the difference in multiple components between instant `a` and instant `b`.
```

Comment:
I think the difference between these need to be more highlighted. Especially `between` vs `inSeconds`. An example could be: `duration.between(2018-01-01, 2018-02-01) = P1M` and `duration.inSeconds(2018-01-01, 2018-02-01) = a duration with a lot of seconds but 0 months`. Applying these durations to a different date, e.g. 2018-02-01, will have different results because of the different lengths of the months.